ebodac/src/main/java/org/motechproject/ebodac/template/PdfReportCTemplate.java

```diff
@@ -11,6 +12,8 @@
 
     public PdfReportCTemplate(OutputStream outputStream) {
         super(TEMPLATE_PATH, FIRST_PAGE_RECTANGLE, outputStream);
+        PdfReportCTemplate.TABLE_FONT = new Font(Font.FontFamily.HELVETICA, 6);
```

Comment:
If you will change the font like this this will affect all reports in ebodac and all entities in booking app.